opre_ops/opre_ops/urls.py

```diff
@@ -0,0 +1,23 @@
+"""opre_ops URL Configuration
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/3.2/topics/http/urls/
+Examples:
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  path('', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+from django.contrib import admin
+from django.urls import path
+from opre_ops.apps.hello_world import views
+
+urlpatterns = [
+    path('admin/', admin.site.urls),
+    path('hello_world/', views.hello_world, name='hello_world')
+]
```

Comment:
There's a bunch of different ways we can do routes in projects with apps with Django, this was just an easy first feature way, not sure how we want to do this in the future though